### PR TITLE
:tada: Pipedrive source: add organizations stream

### LINKED ISSUE #5591: Pipedrive source: Export organisations
## Tell us about the problem you're trying to solve

The Pipedrive connector exports almost every dataset, except organisations for some reason. I would like to have those exported too.

https://developers.pipedrive.com/docs/api/v1/Organizations

## Are you willing to submit a PR?

I'm not familiar enough with connector development and unfortunately don't have the time-

### PR DESCRIPTION
**Pull request by avida**
## What

Resolves #5591

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> New Connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/SUMMARY.md`
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector added to connector index like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
   
</p>
</details>

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</p>
</details>

<details><summary> <strong> Connector Generator </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py [hunk 1]
```diff
@@ -98,10 +98,11 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
...
         records = response.json().get(self.data_field) or []
         for record in records:
-            if record.get(self.data_field):
-                yield record.get(self.data_field)
-            else:
-                yield record
+            record = record.get(self.data_field) or record
+            if self.primary_key in record and record[self.primary_key] is None:
+                # Convert "id: null" fields to "id: 0" since id is primary key and SAT checking if it not null.
+                record[self.primary_key] = 0
```
Comment (htrueman) [inline airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py:104]: You may just set `primary_key=None` and SAT will skip the stream from the pk validation. 
Or is it may be some pks with integer values and some are None?

Comment (avida) [reply airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py:104]: > Or is it may be some pks with integer values and some are None?

Yes this is the case, there is mixed null and integer values for activity_fields stream: https://pastebin.com/0KvSSzYp

File: airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml [hunk 2]
```diff
@@ -41,4 +41,3 @@ tests:
       configured_catalog_path: "integration_tests/non_invoice_line_items_catalog.json"
       timeout_seconds: 3600
     - config_path: "secrets/connected_account_config.json"
-
```
Comment (Zirochkaa) [inline airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml:44]: Why is this file here?

File: airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py [hunk 3]
```diff
@@ -98,10 +98,11 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
         """
         records = response.json().get(self.data_field) or []
         for record in records:
-            if record.get(self.data_field):
-                yield record.get(self.data_field)
-            else:
-                yield record
+            record = record.get(self.data_field) or record
+            if self.primary_key in record and record[self.primary_key] is None:
+                # Convert "id: null" fields to "id: 0" since id is primary key and SAT checking if it not null.
```
Comment (Zirochkaa) [inline airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py:103]:
```suggestion
                # Convert "id: null" fields to "id: 0" since id is primary key and SAT checks if it is not null.
```

**Review by Zirochkaa — APPROVED** [review]
lgtm, just small comments

**Review by htrueman — APPROVED** [review]

File: airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py (hunk 3 above)
Comment (avida) [reply airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py]: updated

File: airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml (hunk 2 above)
Comment (avida) [reply airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml]: reverted

**Review by sherifnada — APPROVED** [review]
